pkg/osm/status.go

```diff
@@ -9,13 +9,12 @@ import (
 func ControlPlaneStatus(osmControlPlaneNamespace string) error {
 	log.Info().Msgf("Determining the status of the OSM control plane in namespace %s", osmControlPlaneNamespace)
 
-	_, err := kuberneteshelper.GetKubeClient()
+	client, err := kuberneteshelper.GetKubeClient()
 	if err != nil {
 		log.Err(err).Msg("Error creating Kubernetes client")
 	}
 
-	// TODO add checks like osm controller log checks
-	outcomes := common.Run()
+	outcomes := common.Run(HasNoBadOsmControllerLogsCheck(client, osmControlPlaneNamespace))
```

Comment:
I think `HasNoBadOsmControllerLogsCheck` could be used in pod-to-pod check... and issue a warning if something is fishy...

Reply:
See https://github.com/openservicemesh/osm-health/pull/50/files#r683763285